filters/locale-date.js

```diff
@@ -0,0 +1,29 @@
+(function () {
+	angular.module('app')
+		.filter('localeDate', ['$translate', localeDate]);
```

Comment:
wouldn't it easier to apply the predefined [date](https://docs.angularjs.org/api/ng/filter/date) filter?